1.9/api/pkgpanda.yaml

```diff
@@ -0,0 +1,207 @@
+swagger: '2.0'
+info:
+  title: DC/OS Pkgpanda API
+  version: '1.0.0'
+basePath: /pkgpanda
+
```

Comment:
@karlkfi let's iterate on the tags later. this shouldn't be a blocker for the initial API doc push.